sdk/python/feast/infra/online_stores/contrib/cassandra_online_store/cassandra_online_store.py

```diff
@@ -88,7 +88,8 @@
         event_ts        TIMESTAMP,
         created_ts      TIMESTAMP,
         PRIMARY KEY ((entity_key), feature_name)
-    ) WITH CLUSTERING ORDER BY (feature_name ASC);
+    ) WITH CLUSTERING ORDER BY (feature_name ASC)
+    {table_options};
```

Comment:
We can use this way when we add support for multiple table options. At this point, can we just change it to `AND default_time_to_live={ttl}`. Pass 0 when ttl is None.